src/internal/m365/collection/site/backup.go

```diff
@@ -247,11 +254,53 @@ func CollectLists(
 		return nil, false, clues.WrapWC(ctx, err, "making metadata collection")
 	}
 
-	spcs = append(spcs, col)
+	spcsMap["metadata"] = col
+
+	for _, spc := range spcsMap {
+		spcs = append(spcs, spc)
+	}
 
 	return spcs, canUsePreviousBackup, el.Failure()
 }
 
+func handleTombstones(
+	ctx context.Context,
+	bpc inject.BackupProducerConfig,
+	tombstones map[string]string,
+	spcsMap map[string]data.BackupCollection,
+	counter *count.Bus,
+	el *fault.Bus,
+) {
+	for id, p := range tombstones {
+		if el.Failure() != nil {
+			return
+		}
+
+		ctx := clues.Add(ctx, "tombstone_id", id)
```

Comment:
```suggestion
		ictx := clues.Add(ctx, "tombstone_id", id)
```

Reply:
let me make the change in the next PR in the chain